cmake/TestCase.cmake

```diff
@@ -4,6 +4,7 @@ macro(test_case)
   list(REMOVE_AT libs 0 0)
 
   add_executable(${trgt} "${trgt}_test.cpp")
```

Comment:
I think this should actually be
```cmake
add_executable(${trgt} $<TARGET_OBJECTS:catch_main> "${trgt}_test.cpp")
```
and remove catch_main from `target_link_libraries`
```

Reply:
yep, that's was right way